libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -169,6 +169,72 @@ bool AP_Arming::logging_checks(bool report)
     return true;
 }
 
+bool AP_Arming::ins_accels_consistent(const AP_InertialSensor &ins)
+{
+    if (ins.get_accel_count() <= 1) {
```

Comment:
Existing behavior I know, but caching the result of "get_accel_count()" would be nice as you can then skip doing the call in the loop. (Same comment applies to the gyro)

Reply:
I'm not sure it's worth it given how often these are called.  However, I've pushed a patch up.